lib/modules/manager/pip-compile/artifacts.ts

```diff
@@ -56,12 +57,18 @@ export function constructPipCompileCmd(
   ) {
     compileArgs.argv.splice(1, 0, '--no-emit-index-url');
   }
+  for (const dep of upgradePackages) {
+    compileArgs.argv.push(
+      `--upgrade-package=${dep.depName}==${dep.newVersion}`,
```

Comment:
how does this need to be quoted?

-  `--upgrade-package='<dep>==<version>'`
-  `'--upgrade-package=<dep>==<version>'`

Reply:
First one. `--upgrade-pacage='dep==1.0.0'`. I also extracted this change to https://github.com/renovatebot/renovate/pull/27353. As this can be treated as a standalone feature for normal `update-lockfile`. Sorry for the confusion.